### PR TITLE
mycolumns environment cannot be nested within itself

### PR DESCRIPTION
**Pull request by EagleoutIce**
This seems to be bug within pgfkeys. The problem seems to be that `width` is a name that is found by `\pgfkeysalso` (e.g. there is another pgf keys named `with`). So now we are using the defaults of the surrounding column environments which seems to work well:

Screenshot_20221028_124552

### CODE REVIEW

File: demo-slides/demoSlides.tex [hunk 1]
```diff
@@ -172,7 +172,42 @@ \subsection{Columns with Custom Height}
...
+			\mynote{}{1.4}
+		\end{mycolumns}
+	\mynextcolumn
+		\begin{example}{}
+			A column on the right side
+		\end{example}
+		\begin{mycolumns}
+			\mynote{}{2.1}
+			\begin{mycolumns}[columns=3]
+				\mynote{}{1}
```
Comment (Benno2000) [inline demo-slides/demoSlides.tex:201]:
```suggestion
				\mynote{}{2.1.1}
```

File: demo-slides/demoSlides.tex [hunk 2]
```diff
@@ -172,7 +172,42 @@ \subsection{Columns with Custom Height}
...
+				\mynextcolumn
+				\mynote{}{2}
```
Comment (Benno2000) [inline demo-slides/demoSlides.tex:203]:
```suggestion
				\mynote{}{2.1.2}
```

File: demo-slides/demoSlides.tex [hunk 3]
```diff
@@ -172,7 +172,42 @@ \subsection{Columns with Custom Height}
...
+				\mynextcolumn
+				\mynote{}{3}
```
Comment (Benno2000) [inline demo-slides/demoSlides.tex:205]:
```suggestion
				\mynote{}{2.1.3}
```

**Review by Benno2000 — CHANGES_REQUESTED** [review]
Works very well for me! I would change the deepest nested numbers in the demo slides though to make the example more understandable.

**Comment by EagleoutIce** [comment]
I had the numbers you suggested before pushing. But the resulting boxes are too small: 

Screenshot_20221109_112641

If you want, i can still add the requested changes.

**Review by Benno2000 — APPROVED** [review]

**Pull request merged**